VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/private-consultation-participants/private-consultation-participants.component.html

```diff
@@ -1,13 +1,20 @@
 <div class="container">
     <h2 class="govuk-heading-s white">{{ 'private-consultation-participants.participants' | translate }}</h2>
     <div class="participant-grid">
-        <ng-container *ngFor="let participant of getPrivateConsultationParticipants(); trackBy: trackParticipant">
+        <ng-container *ngFor="let participant of getPrivateConsultationParticipants() trackBy: trackParticipant">
             <div class="language-icon"
                 *ngIf="participant.interpreter">
                 <span class="far fa-language"></span>
             </div>
-            <div class="participant-row">
-                <div [class]="getRowClasses(participant)">
+            <div  class="language-icon" *ngIf="isJohConsultation() && isFirstJohFromGroupMember(participant)">
```

Comment:
Personally if grouping is important I would consider grouping by html in some way. This feels more semantically correct and may have an impact on accessibility(e.g. screen readers). This may be something to discuss with UX guys but will assume css will suffice for this review.

Reply:
grouping done in next commit